docs/examples/add_data_to_dataset.ts

```diff
@@ -1,10 +1,10 @@
-import { Dataset, CheerioCrawler } from 'crawlee';
+import { CheerioCrawler } from 'crawlee';
 
 const crawler = new CheerioCrawler({
     // Function called for each URL
     async requestHandler({ request, body }) {
         // Save data to default dataset
-        await Dataset.pushData({
+        await crawler.pushData({
```

Comment:
This is better but still not optimal - you want to use `context.pushData`

Reply:
just so its clear, you want this instead:

```ts
    async requestHandler({ request, body, pushData }) {
        // Save data to default dataset
        await pushData({
```